Src/DependencyCollector/Shared/Implementation/HttpParsers/DocumentDbHttpParser.cs

```diff
@@ -0,0 +1,196 @@
+            ["GET dbs/*/colls/*/docs/*/attachments"] = "List Attachments",
+            ["GET dbs/*/colls/*/docs/*/attachments/*"] = "Get Attachment",
+            ["DELETE dbs/*/colls/*/docs/*/attachments/*"] = "Delete Attachment",
+            ["PUT dbs/*/colls/*/docs/*/attachments/*"] = "Replace Attachment",
+            // Stored procedure operations
+            ["POST dbs/*/colls/*/sprocs"] = "Create Stored Procedure",
+            ["PUT dbs/*/colls/*/sprocs/*"] = "Replace Stored Procedure",
+            ["GET dbs/*/colls/*/sprocs"] = "List Stored Procedures",
+            ["DELETE dbs/*/colls/*/sprocs/*"] = "Delete Stored Procedure",
+            ["POST dbs/*/colls/*/sprocs/*"] = "Execute Stored Procedure",
+            // User defined function operations
+            ["POST dbs/*/colls/*/udfs"] = "Create UDF",
+            ["PUT dbs/*/colls/*/udfs/*"] = "Replace UDF",
+            ["GET dbs/*/colls/*/udfs"] = "List UDFs",
+            ["DELETE dbs/*/colls/*/udfs/*"] = "Delete UDF",
+            // Trigger operations
+            ["POST dbs/*/colls/*/triggers"] = "Create Trigger",
+            ["PUT dbs/*/colls/*/triggers/*"] = "Replace Trigger",
+            ["GET dbs/*/colls/*/triggers"] = "List Triggers",
+            ["DELETE dbs/*/colls/*/triggers/*"] = "Delete Trigger",
+            // User operations
+            ["POST dbs/*/users"] = "Create User",
+            ["GET dbs/*/users"] = "List Users",
+            ["GET dbs/*/users/*"] = "Get User",
+            ["DELETE dbs/*/users/*"] = "Delete User",
+            ["PUT dbs/*/users/*"] = "Replace User",
+            // Permission operations
+            ["POST dbs/*/users/*/permissions"] = "Create Permission",
+            ["GET dbs/*/users/*/permissions"] = "List Permission",
+            ["GET dbs/*/users/*/permissions/*"] = "Get Permission",
+            ["DELETE dbs/*/users/*/permissions/*"] = "Delete Permission",
+            ["PUT dbs/*/users/*/permissions/*"] = "Replace Permission",
+            // Offer operations
+            ["POST offers"] = "Query Offers",
+            ["GET offers"] = "List Offers",
+            ["GET offers/*"] = "Get Offer",
+            ["PUT offers/*"] = "Replace Offer"
+        };
+
+        /// <summary>
+        /// Tries parsing given dependency telemetry item. 
+        /// </summary>
+        /// <param name="httpDependency">Dependency item to parse. It is expected to be of HTTP type.</param>
+        /// <returns><code>true</code> if successfully parsed dependency.</returns>
+        internal static bool TryParse(ref DependencyTelemetry httpDependency)
+        {
+            string name = httpDependency.Name;
+            string host = httpDependency.Target;
+            string url = httpDependency.Data;
+
+            if (name == null || host == null || url == null)
+            {
+                return false;
+            }
+
+            if (!host.EndsWith(".documents.azure.com", StringComparison.OrdinalIgnoreCase))
```

Comment:
Just found  that we dont do it for blob/queue etc. :(
https://github.com/Microsoft/ApplicationInsights-dotnet-server/blob/2.4.0-beta1/Src/DependencyCollector/Shared/Implementation/HttpParsers/AzureBlobHttpParser.cs

need a task to do it as well.